src/algorithms/fardetectors/FarDetectorLinearTrackingConfig.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023, Simon Gardner
+
+#pragma once
+
+#include <spdlog/spdlog.h>
+#include "FarDetectorTrackerClusterConfig.h"
+
+namespace eicrecon {
+  struct FarDetectorLinearTrackingConfig {
+
+    FarDetectorTrackerClusterConfig detconf;
```

Comment:
Nesting config is not allowed. That prevents specifying configuration in a simple format where we don't need to know the exact structure underlying this. Each config should be simple types only.